package.json

```diff
@@ -247,7 +247,7 @@
     "url-search-params": "^0.10.0",
     "us-forms-system": "https://github.com/usds/us-forms-system.git#f476ed0ff038403d04e531bbbae41f72290e23b5",
     "vanilla-lazyload": "^8.17.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#92816fb3a3d5166dba1404e14c2e12739ba3b3b2",
+    "vets-json-schema": "department-of-veterans-affairs/vets-json-schema.git#v3.119.0",
```

Comment:
If `vets-json-schema` gets updated before this gets merged in, I'll update this to match the new version.